apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue

```diff
@@ -171,15 +172,15 @@ async function regenerate() {
   }
 }
 
-function onMessageEntered({ message, event }: DxChatTypes.MessageEnteredEvent) {
+function onMessageEntered({ message, event }: DxChatTypes.MessageEnteredEvent): void {
   dataSource.store().push([{ type: 'insert', data: { id: Date.now(), ...message } }]);
 
-  if (!alerts.value.length) {
+  if (!alerts.value.length && event) {
```

Comment:
This is not necessary because the event param can be undefined.

```suggestion
  if (!alerts.value.length) {
```